Add Lighter

amine1107 opened the pull request:
Add Lighter: Lighter is a flexible Css micro framework,Based on the visual aspect, the framework offers you a clean and easy work.

amine1107 committed:
Update README.md

amine1107 committed:
Update Readme

Add Lighter 
Add Siga

youknowriad commented on README.md:
```diff
@@ -24,12 +24,14 @@ A curated list of awesome things and projects built by Algerian developpers.
   - [Nightly.js](https://github.com/Fcmam5/nightly.js) - A zero dependency JavaScript library for enabling the Dark mode in your UI
   - [react-graphql-redux](https://github.com/youknowriad/react-graphql-redux) - use GraphQL to query your Redux store
   - [Rungen](https://github.com/youknowriad/rungen) - A generic runtime around async flow in javascript
+  - [Siga](https://github.com/amine1107/siga) - JavaScript Library to make development in javascript easier. 
```
This one needs a license and maybe a little bit better README to explain how to use it (maybe just a link to the docs). ;)

amine1107 replied:
the documentation is on github page,and I'm working on readme

amine1107 committed:
Adding Lighter Css Framework

I'm working on a good clean siga documentation so I decided to delete it from readme and adding Lighter

youknowriad commented on README.md:
```diff
@@ -24,6 +24,7 @@ A curated list of awesome things and projects built by Algerian developpers.
 ## Libraries and Frameworks
 - CSS
   - [Hexa](https://github.com/amine1107/Hexa) A simple and lightweight CSS animation library
+  - [Lighter](https://github.com/amine1107/lighter) Lighter is a flexible Css micro framework based on the visual aspect, the framework offers you a clean and easy work.
```
Typo `Css` => `CSS`

amine1107 replied:
i don't understand what you mean

youknowriad replied:
Instead of `Css`, can you write `CSS`?

amine1107 committed:
Update README.md

youknowriad reviewed (approved):
Thanks for the update
